### ISSUE TITLE
Latest package does not include root imports.

### ISSUE THREAD

**Issue by DSchroer** [PlayFab/NodeSDK#105]
The latest package version made some changes to the `package.json` file. It now only includes scripts from within the `Scripts` directory. This breaks any code that made use of:
```js
const pf = require("playfab-sdk");
```

**Comment by DSchroer** [comment]
The change was introduced with this commit:
https://github.com/PlayFab/NodeSDK/commit/e1449de85e132424c79d79cddad9fe142528c0ac

**Comment by najadojo** [comment]
Please this address this issue it makes this not usable with TypeScript.

**Comment by lazerwalker** [comment]
Wanted to +1 this, this is a _serious_ issue for anyone using PlayFab using TypeScript with Node.

For what it's worth, the generated SDK by the https://github.com/PlayFab/SDKGenerator repo is still including index.js (as evidenced by this repo, but I also confirmed by running the generator myself). The problem seems to be with what happens when PlayFab's automated processes publish to npm, and those processes/scripts seems to not be public code.

**For anyone else running into this, I have a (hacky) workaround**. Because this repo has a valid copy of the SDK, you can point npm here instead. However, because the package itself lives in a subdirectory, you can't just point npm at this git repo.

Add this postinstall script to your `package.json` file, and it should work.

```js
scripts: {
    "postinstall": "mkdir playfab-sdk; cd playfab-sdk; git init; git remote add -f origin https://github.com/PlayFab/NodeSDK.git; git config core.sparseCheckout true; echo \"PlayFabSddkK\" >> .git/info/sparse-checkout; git pull --depth=1 origin master; cd ..; npm i ./playfab-sdk/PlayFabSdk/"
    ...
}
````

Short version: this creates a `playfab-sdk` folder in the root of your project, which contains a clone of this git repo that JUST has the `PlayFabSdk` subdirectory. From there, it installs the npm module located in that local folder, which will be the valid package located at https://github.com/PlayFab/NodeSDK/tree/master/PlayFabSdk.

**Comment by lazerwalker** [comment]
If you don't need new changes that have gone in the past few weeks, you can also pin your npm version to `2.43.200303` and it should work.

I'd imagine `2.44.200330` might work as well, but I haven't tested.

**Comment by pgilmorepf** [comment]
Thanks for the bug report, and thanks to @lazerwalker for sending us the fix.
Release: https://github.com/PlayFab/NodeSDK/releases/tag/2.46.200422
Release Notes: https://docs.microsoft.com/en-us/gaming/playfab/release-notes/#200422
NPM Package: https://www.npmjs.com/package/playfab-sdk
